.travis.yml

```diff
@@ -58,8 +58,8 @@ script:
 - gradle test -b test/java/build.gradle
 - go test -coverprofile=coverage.txt -covermode=atomic github.com/anz-bank/sysl/...
 - npm test --prefix sysl2/sysl/sysl_js
-- GOOS=darwin GOARCH=amd64 go build -o gosysl/gosysl-darwin github.com/anz-bank/sysl/sysl2/sysl
-- GOOS=linux GOARCH=amd64 go build -o gosysl/gosysl-linux github.com/anz-bank/sysl/sysl2/sysl
+- GOOS=darwin GOARCH=amd64 go build -o gosysl/gosysl-darwin -ldflags "-X main.Version=${TRAVIS_TAG:1}" github.com/anz-bank/sysl/sysl2/sysl
+- GOOS=linux GOARCH=amd64 go build -o gosysl/gosysl-linux -ldflags "-X main.Version=${TRAVIS_TAG:1}" github.com/anz-bank/sysl/sysl2/sysl
```

Comment:
Needs a test, maybe add one to `test-gosysl.sh`. Since you can't predict the version number, just check that `--version` works.

Reply:
Done